include/tvm/relay/attrs/transform.h

```diff
@@ -78,7 +78,7 @@ struct InitOpAttrs : public tvm::AttrsNode<InitOpAttrs> {
       .describe("Target shape.");
     TVM_ATTR_FIELD(dtype)
       .describe("Target data type.")
-      .set_default(Int(0));
+      .set_default(NullValue<DataType>());
```

Comment:
Thank you for making this change. One more small request: can you make sure that `FullRel()` in src/relay/op/tensor/transform.cc is still correct? It checks the number of bits in the data type to see if it should be treated as the default, which I'm not sure would play correctly with `NullValue<DataType>()` (this should be covered by the tests, but maybe this isn't idiomatic code even if it happens to work).

Reply:
It is, nullvalue databtype returns a type with bits equals 0